### PR TITLE
Updating Desc and Typo

### PR DESCRIPTION
**Pull request by Ikhari**
According to and from :
https://github.com/AdguardTeam/AdGuardHome#getting-started

## Type of change

- [X ] Spelling, grammar or other readability improvements (`current` branch).
- [ X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Ikhari, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19652).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/adguard.markdown [hunk 1]
```diff
@@ -16,9 +16,10 @@ ha_platforms:
   - switch
 ---
 
-AdGuard Home is a network-wide ad- and tracker-blocking DNS server with parental
-control (adult content blocking) capabilities. The AdGuard integration allows
-you to control and monitor your AdGuard Home instance in Home Assistant.
+AdGuard Home is a network-wide software for blocking ADS & tracker-blocking DNS server with parental
+control (adult content blocking) capabilities.
+After you set it up, it'll cover ALL your home devices, and you don't need any client-side software for that.
```
Comment (frenck) [inline source/_integrations/adguard.markdown]:
```suggestion
After you set it up, it'll cover ALL your home devices, and you don't need any client-side software for that.
```

This is misleading, as this is not AdGuard Home itself, but the integration.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/adguard.markdown (hunk 1 above)
Comment (Ikhari) [reply source/_integrations/adguard.markdown]: Done, sorry about that.

Comment (frenck) [reply source/_integrations/adguard.markdown]: This line is language-wise incorrect, the grammar is not right and the flow is not correct (blocking blocking)

Comment (Ikhari) [reply source/_integrations/adguard.markdown]: check, if you still don't like it, just close it

**Comment by Ikhari** [comment]
check

File: source/_integrations/adguard.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/adguard.markdown]: It again contains typos. For example, first sentence ends with a full stop, while the next line doesn't start with a capital letter. The last sentence has a space before its full stop. `Protecting` is capitalized, devices should be plural.

Looking what you are trying to change versus what is already there, I don't see the need for this change.

Therefore I'm going to friendly decline this change request. Nevertheless, thanks for willing to contribute 👍

**Pull request closed without merging**